_i18n/en/general/license.md

```diff
@@ -1,4 +1,8 @@
-# Jekyll theme
+# AntennaPod
+
+AntennaPod is licensed under the [General Public License](https://github.com/AntennaPod/AntennaPod/blob/develop/LICENSE).
```

Comment:
Hmm, I wouldn't translate the license and other legal stuff (privacy policy etc)

Reply:
Yeah, I was hesitant about this one. I'll keep only the non-legal parts translatable. Our 'privacy policy' can be fully translated. I strongly doubt it has any legal value, and explains what data other parties have access to.

Reply:
Hmm, it should try to have legal value. At least Google requires adding a privacy policy and we link it.

Reply:
Sure, we try, but we don't have any legal lingo that might be lost in translation, which could have legal repercussions. We're simply explaining what AntennaPod does in layman's terms. Having such explanation in the user's language would be helpful to the user. If you want, we can add a sentence like 'in case of unclarity or inconsistencies between language versions, the English text will prevail' ([examples](https://www.lawinsider.com/clause/prevailing-language)).

Reply:
Yeah, I think adding such a sentence would be nice